tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/QuietBackgroundProcessesView.xaml

```diff
@@ -7,7 +7,7 @@
     Loaded="UserControl_Loaded">
 
     <StackPanel Orientation="Vertical"
-                Visibility="{x:Bind ViewModel.IsQuietBackgroundProcessesFeatureEnabled, Mode=OneWay}">
```

Comment:
>IsQuietBackgroundProcessesFeatureEnabled [](http://example.com/codeflow?start=46&length=40)

this fixes showing for Win 10, but now you've lost the IsQuietBackgroundProcessesFeatureEnabled check.

the feature will always show, even if the toggle is switched off in Settings -> Experimental features.

    public bool IsQuietBackgroundProcessesFeatureEnabled => _experimentationService.IsFeatureEnabled("QuietBackgroundProcessesExperiment");

Reply:
IsFeaturePresent is only ever set to true if IsQuietBackgroundProcessesFeatureEnabled is true.

Reply:
oh snap you are right